bundle/src/bundle_meta.rs

```diff
@@ -83,5 +93,35 @@ pub enum VersionedBundle {
     V0_6_2(BundleMetaV0_6_2),
 }
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+#[cfg(feature = "pyo3")]
+#[gen_stub_pyclass]
+#[pyclass]
+pub struct BindingsVersionedBundle(pub VersionedBundle);
+
+#[cfg(feature = "pyo3")]
+#[gen_stub_pymethods]
+#[pymethods]
+impl BindingsVersionedBundle {
+    pub fn get_v0_5_29(&self) -> Option<BundleMetaV0_5_29> {
+        match &self.0 {
+            VersionedBundle::V0_5_29(bundle_meta) => Some(bundle_meta.clone()),
+            _ => None,
+        }
+    }
+    pub fn get_v0_5_34(&self) -> Option<BundleMetaV0_5_34> {
+        match &self.0 {
+            VersionedBundle::V0_5_34(bundle_meta) => Some(bundle_meta.clone()),
+            _ => None,
+        }
+    }
+    pub fn get_v0_6_2(&self) -> Option<BundleMetaV0_6_2> {
+        match &self.0 {
+            VersionedBundle::V0_6_2(bundle_meta) => Some(bundle_meta.clone()),
+            _ => None,
+        }
+    }
+}
```

Comment:
Using `gen_stub_pymethods`doesn't seem to work on enums yet:
https://github.com/Jij-Inc/pyo3-stub-gen/issues/69

All that's needed is a new type here to get around it

Also, the Python bindings aren't using `serde`, so `VersionedBundle` doesn't end up creating a nice `dict`—we can just use methods to get the `meta.json` values